src/shared/journal-importer.c

```diff
@@ -69,9 +69,9 @@ static int get_line(JournalImporter *imp, char **line, size_t *size) {
 
                 imp->scanned = imp->filled;
                 if (imp->scanned >= DATA_SIZE_MAX)
-                        return log_error_errno(SYNTHETIC_ERRNO(ENOBUFS),
-                                               "Entry is bigger than %u bytes.",
-                                               DATA_SIZE_MAX);
+                        return log_warning_errno(SYNTHETIC_ERRNO(ENOBUFS),
+                                                 "Entry is bigger than %u bytes.",
```

Comment:
add ", ignoring"?

Reply:
The caller decides what to do... So I don't think we should say "ignoring" here.